subgraph/subgraph.yaml

```diff
@@ -5,7 +5,7 @@ dataSources:
 - kind: ethereum/contract
   name: Bitsausage
   source:
-    address: "AeB9Ad0EaeE1Ea1B47f181c8C2e7b5927b25106c"
+    address: "0xF0909E9C610dC497260eB0729D181275251B467b"
```

Comment:
i think we are in a prediciment here. since auctions get launched manually

In sausagetokens.sol, line 31, i have a commented out function that long term should work. becuase if we log all auctions created through the sausage token contract, we can only follow that one sausage token contract, and get all events of all the Auctions made. Right now I think we can only follow one auction, and subgraphs yamls have to be hardcoded in . so we need the SausageToken.sol file to be an auction factory